.gitignore

```diff
@@ -1,3 +1,4 @@
 output
 __pycache__/
 *.log
+dellemc-powermax-*.tar.gz
```

Comment:
All the tar.gz will be removed in my PR https://github.com/dell/ansible-powermax/pull/65/files

Reply:
removed